creator/data_templates/queries/template_version.py

```diff
@@ -0,0 +1,39 @@
+from pprint import pprint
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphql import GraphQLError
+from django_filters import FilterSet, OrderingFilter, CharFilter
+
+from creator.data_templates.nodes.template_version import TemplateVersionNode
+from creator.data_templates.models import TemplateVersion
+
+
+class TemplateVersionFilter(FilterSet):
+    order_by = OrderingFilter(fields=("created_at", "modified_at"))
+    study_kf_id = CharFilter(field_name="studies__kf_id", lookup_expr="exact")
```

Comment:
I think it would be preferable if we use relay ids here as we use them almost exclusively from the front end since it allows cleaner cache management.